src/main.rs

```diff
@@ -1,2 +1,42 @@
-fn main() {
+use warp::Filter;
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+
+use geoengine_operators::Operator;
+
+
+#[tokio::main]
+async fn main() {
+    pretty_env_logger::init();
+
+    let workflow_registry = Arc::new(Mutex::new(HashMap::<usize, Operator>::new()));
+    let workflow_registry = warp::any().map(move || Arc::clone(&workflow_registry));
```

Comment:
Das ist irgendwie verwirrend, dass du zweimal die gleiche Variable direkt untereinander deklarierst.
Vielleicht einmal deklarieren und dann den "Kram" in einem inneren Block?

Die obere Variable ist weg, kann man dann nur noch via `warp` auf die Registry zugreifen?